packages/ui/src/kit/VRadio/VRadio.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <label :class="vRadioCssClass">
+    <input
+      v-bind="$attrs"
+      :checked="value === modelValue"
+      type="radio"
+      class="v-radio-input"
+      @input="onInputChange"
+    />
+    <span class="v-radio-box"></span>
+    <span v-if="$slots.default" class="v-radio-label">
+      <slot />
+    </span>
+  </label>
+</template>
+
+<script>
+import ThemeMixin from '@/mixins/ThemeMixin';
+
+export default {
+  name: 'VRadio',
+
+  props: {
+    value: {
+      type: String,
+      required: true,
+    },
+
+    modelValue: {
+      type: String,
+      default: null,
+    },
+
+    isError: {
+      type: Boolean,
+      default: false,
+    }
+  },
+
+  computed: {
+    vRadioCssClass() {
+      return Object.assign({}, this.themeCssClass, {
+        'v-radio': true,
+        'is-error': this.isError,
+        'is-disabled': this.$attrs.disabled,
+      });
+    },
+  },
+
+  methods: {
+    onInputChange(e) {
+      this.$emit('input', this.value);
```

Comment:
In [that chapter](https://vuejs.org/v2/guide/forms.html) - 'checkboxes and radiobuttons use checked property and change event' and we want to use as 'v-model'